### DISCUSSION TITLE
react native dynamic links

### DISCUSSION THREAD

**Discussion by mohamed000123** [invertase/react-native-firebase#6647] [Q&A]
i am using react native dynamic links and i have this error
can someone help please 
Error: You attempted to use a firebase module that's not installed natively on your iOS project by calling firebase.dynamicLinks(). Ensure you have either linked the module or added it to your projects Podfile.

**Comment by mikehardy** [comment]
The error message is pretty descriptive. You haven't given any other information to go on so help is not really possible
https://stackoverflow.com/help/how-to-ask

Run this, and you'll see the resulting project shows you how to integrate everything https://github.com/mikehardy/rnfbdemo/blob/main/make-demo.sh

But you might just have forgotten to add the package *then pod install*
